package.json

```diff
@@ -13,13 +13,13 @@
   "author": "Jose Martinez",
   "license": "MIT",
   "dependencies": {
-    "@capacitor/core": "^3.4.3"
+    "@capacitor/core": "^5.0.3"
   },
   "devDependencies": {
-    "@capacitor/android": "^3.4.3",
-    "@capacitor/ios": "^3.4.3",
-    "rimraf": "^3.0.0",
-    "typescript": "^4.2.4"
+    "@capacitor/android": "^5.0.3",
+    "@capacitor/ios": "^5.0.3",
+    "rimraf": "^5.0.1",
```

Comment:
I'd stay with rimraf ^4.4.1 because capacitor itselfs references ^4.4.1.
Thank you for your contribution!